src/components/Illustration/illustrations/PraiseGiftSmaller72.tsx

```diff
@@ -1,29 +0,0 @@
-/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
-/* tslint:disable:max-line-length */
-// This file was auto-generated -- do not edit
-
-import * as React from 'react';
-import BaseIllustration from '../BaseIllustration';
-
-export default class PraiseGiftSmaller extends BaseIllustration {
-  public render() {
-    const { size = 72 } = this.props;
-
-    return (
-      <svg
-        viewBox="0 0 72 72"
-        className={this.getClassName(size)}
-        aria-hidden="true"
-        tabIndex={-1}
-        focusable="false"
-      >
-        <circle cx="-35.231" cy="36.753" r={36} fill="#386cbb" transform="rotate(-88.789)" />
-  <path fill="#ffc56c" d="M40.653 23.425l-5.136-10.032c-1.2-2.352-4.368-2.736-6.144-.816a3.76 3.76 0 0 0 1.152 5.904z" />
-  <path fill="#ffc56c" d="M32.445 23.425l5.136-10.032c1.2-2.352 4.368-2.736 6.144-.816a3.76 3.76 0 0 1-1.152 5.904z" />
-  <path fill="#fb7f78" d="M49.245 58.08H23.758c-3.264 0-5.904-2.592-6.048-5.808l-1.008-26.015a6.02 6.02 0 0 1 6.048-6.288H50.3c3.456 0 6.192 2.88 6.048 6.288l-1.008 26.015c-.144 3.264-2.832 5.808-6.096 5.808z" />
-  <path fill="#ffc56c" d="M55.677 41.76h-38.4l-.24-6.575h38.976z" />
-  <path fill="#ffc56c" d="M33.31 19.873h6.527V57.84H33.31z" />
-      </svg>
-    );
-  }
-}
```

Comment:
Ah, it looks like this file wasn't in the `assets/` folder, so it got removed. @Logrologist can you confirm it's not needed?

Reply:
it looks like this file was generated then the asset renamed all in the original PR, so it's correct that it should be removed